cardano-wallet.cabal

```diff
@@ -47,6 +53,7 @@ library
     , text
     , time-units
     , transformers
+    , QuickCheck
```

Comment:
No QuickCheck / Arbitrary instances in the source please :pray: 
Those only makes sense during testing.

Reply:
corrected